### ISSUE TITLE
Feature: Members

### ISSUE THREAD

**Issue by xla** [radicle-dev/radicle-upstream#349]
Outstanding work for member management.

**proxy**

- [x] list members for org
- [x] add member endpoint
- [x] ~remove member endpoint~ (later feature)
- [x] refactor user org list to filter by handle instead of the author key

**proxy <> ui**
- [x] use registration endpoint when submitting the transaction in the modal
- [x] validation: do not allow registering an existing member

**ui**

- [ ] add member flow
  - [x] first member flow iteration ( #334 )
  - [ ] show avatar/imageURL of user in text input
- [ ] member list
  - [x] first member list iteration ( #402 )
  - [ ] navigate to user page
  - [x] use actual member list
- [x] ensure correct handling of member txs in transaction center
- [x] thorough testing: additional users currently can only be created in the testing environment

Part of #277

**Comment by MeBrei** [comment]
### Open tasks

1. **Navigate to the user page from the org member list** it is currently not possible in the ui to show the user page of a different user than stored in the session. As far as I can see, there is also no figma screen for a "different" user. So should this be postponed and remain a console log for now?
2. **Copying the id of the member** Currently only the `User` is returned for the org member list. This is waiting on the decisions/implementation of partial user data and how we will connect User and Identity data.
3. **Show avatar/imageURL of the user** It should be shown in the member registration modal and the members list. This is waiting on the decisions/implementation of partial user data and how we will connect User and Identity data as well.
4. **Validating if the user is already a member of the org** This could be added as a validation in the ui as a first version, but validations will move into the proxy in the long run #166 

The action dropdown for the individual members in the org member list currently differs from the one shown in figma, I would argue changing it once the identity and navigation is in place.

**Comment by cloudhead** [comment]
> Navigate to the user page from the org member list

As you noted, this isn't designed, and so is out of scope of this feature.
We *are* working on user profiles, which is part of `Identities 1.` and we're hoping to get in the B1 release. But I would consider that a different feature.

**Comment by MeBrei** [comment]
Closing this issue, as the remaining **open tasks** are actually part of other features and problems.
